eventmesh-security-plugin/eventmesh-security-auth-token/src/main/java/org/apache/eventmesh/auth/token/impl/auth/AuthTokenUtils.java

```diff
@@ -138,14 +74,44 @@ public static boolean authAccess(AclProperties aclProperties) {
         String topic = aclProperties.getTopic();
 
         Object topics = aclProperties.getExtendedField("topics");
-
+      
         if (!(topics instanceof Set)) {
```

Comment:
Please remove these redundant whitespaces here.

Reply:
Thanks for your review, will do :)

Reply:
Is this PR ready for review now?😊

Reply:
Yes, if I am not mistaken :)